File: Work/WK/MS/Web_WebSV/Web/Web.Core/wwwroot/CTS/ClientApp/yinsiang-lottery/src/models/PlayTypeInfo.ts
import { PlayTypeRadioInfo } from "./PlayTypeRadioInfo";

export interface PlayTypeInfo {
    basePlayTypeId: number
    info: {
        lotteryID: number
        playTypeID: number
        playTypeName: string
        userType: number
    }
    playTypeEnum: string
    playTypeRadioInfos: { [key: string]: PlayTypeRadioInfo[] }
}

File: Work/WK/MS/Lib/JxBackendServiceLibrary/JxBackendServiceStd/Model/Entity/Game/Lottery/PlayTypeInfo.cs
using System.ComponentModel.DataAnnotations;

namespace JxBackendService.Model.Entity.Game.Lottery
{
    public class PlayTypeInfo
    {
        [Key]
        public int PlayTypeID { get; set; }

        public string PlayTypeName { get; set; }

        public int LotteryID { get; set; }

        public int Priority { get; set; }

        public int Status { get; set; }

        public int UserType { get; set; }
    }
}